docs/self-managed/setup/deploy/azure/microsoft-aks/aks-helm.md

```diff
@@ -0,0 +1,328 @@
+
+<details>
+<summary>Show configuration to enable internal Elasticsearch</summary>
+
+```yaml
+global:
+  elasticsearch:
+    enabled: true
+  opensearch:
+    enabled: false
+
+elasticsearch:
+  enabled: true
+```
+
+</details>
+
+#### Use internal PostgreSQL instead of the managed PostgreSQL service
+
+If you prefer not to use the external Azure Database for PostgreSQL service, you can switch to the internal PostgreSQL deployment. In this case, configure the Helm chart as follows and remove certain configurations related to the external database and secret references:
+
+<details>
+<summary>Show configuration changes to disable external database usage</summary>
+
+```yaml
+identityKeycloak:
+  postgresql:
+    enabled: true
+
+  # Remove external database configuration
+  # externalDatabase:
+  #   ...
+
+  # Remove service account and annotations
+  # serviceAccount:
+  #   ...
+
+  # Remove extra environment variables for external database driver
+  # extraEnvVars:
+  #   ...
+
+postgresql:
+  enabled: true
+
+webModeler:
+  # Remove this part
+
+  # restapi:
+  #     externalDatabase:
+  #         url: jdbc:postgresql://$\{DB_HOST}:5432/$\{DB_WEBMODELER_NAME}
+  #         user: $\{DB_WEBMODELER_USERNAME}
+  #         existingSecret: webmodeler-postgres-secret
+  #         existingSecretPasswordKey: password
+
+identity:
+  # Remove this part
+
+  # externalDatabase:
+  #     enabled: true
+  #     host: $\{DB_HOST}
+  #     port: 5432
+  #     username: $\{DB_IDENTITY_USERNAME}
+  #     database: $\{DB_IDENTITY_NAME}
+  #     existingSecret: identity-postgres-secret
+  #     existingSecretPasswordKey: password
+```
+
+</details>
+
+#### Fill your deployment with actual values
+
+Once you've prepared the `values.yml` file, run the following `envsubst` command to substitute the environment variables with their actual values:
+
+```bash reference
+https://github.com/camunda/camunda-deployment-references/blob/main/generic/kubernetes/single-region/procedure/assemble-envsubst-values.sh
+```
+
+Next, store various passwords in a Kubernetes secret, which will be used by the Helm chart. Below is an example of how to set up the required secret. You can use `openssl` to generate random secrets and store them in environment variables:
+
+```bash reference
+https://github.com/camunda/camunda-deployment-references/blob/main/generic/kubernetes/single-region/procedure/generate-passwords.sh
+```
+
+Use these environment variables in the `kubectl` command to create the secret.
+
+- The `smtp-password` should be replaced with the appropriate external value ([see how it's used by Web Modeler](/self-managed/modeler/web-modeler/configuration/configuration.md#smtp--email)).
```

Comment:
This is also a bit odd. smpt is needed for Web Modeller, but we are not configured by default it.